### DISCUSSION TITLE
Inconsistency in errorCode and statusCode names

### DISCUSSION THREAD

**Discussion by bjuriewicz** [jsforce/jsforce#1473] [Ideas]
When parsing the errors from `allOrNone=false` the error code string lives in `statusCode` property of each records' error object. On the other hand, when doing some 'single result' operation like select, the error code lives in `errorCode` property, which might be a bit confusing, when looking for `INVALID_SESSION_ID` code for example.

**Comment by cristiand391** [comment]
triaging old issues...

in both cases mentioned jsforce is returning the errors as they come from the API.

example with allOrNone=true trying to delete 2 accounts (one invalid ID):

```
➜  jsforce git:(cd/bulk-add-missing-prop) ✗ sf org api 'services/data/v50.0/composite/sobjects?ids=0015200001CX73xAAD,0010500000fxR1FAAU&allOrNone=true' --method DELETE
[
  {
    "id": "0015200001CX73xAAD",
    "success": false,
    "errors": [
      {
        "statusCode": "ALL_OR_NONE_OPERATION_ROLLED_BACK",
        "message": "Record rolled back because not all records were valid and the request was using AllOrNone header",
        "fields": []
      }
    ]
  },
  {
    "success": false,
    "errors": [
      {
        "statusCode": "INVALID_CROSS_REFERENCE_KEY",
        "message": "invalid cross reference id",
        "fields": []
      }
    ]
  }
]
```

moving to GH discussions get more feedback about error standarization.
